packages/danger/babel.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('expo-module-scripts/babel.config.base');
```

Comment:
I think it should have been generated by `expo-module-scripts` 🤔 Then it has a comment on top of the file.